sparkctl/README.md

```diff
@@ -101,6 +101,18 @@ Usage:
 $ sparkctl status <SparkApplication name>
 ```
 
+### Events
+
+`events` is a sub command of `sparkctl` for streaming `SparkApplication` events in the namespace 
+specified by `--namespace`. The command will display events since last creation of the `SparkApplication`
+for the specific `name`, and continues to stream events even 
+if `ResourceVersion` or `UID` changes due to completion or deletion of `SparkApplication`.
```

Comment:
I don't think `UID` will change for the lifetime of a `SparkApplication` object. And does it make sense to continue streaming events even if the `SparkApplication` gets deleted?

Reply:
The problem I faced is that the stream after deletion event still continues. So I would need to check for "SparkApplicationDeleted" event so streaming terminates, or it will wait forever (till timeout)